CrPatterns/src/main/java/Singelton/ConnectionSingelton.java

```diff
@@ -0,0 +1,39 @@
+package Singelton;
+
+
+import abstrFabr.constants.Constants;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+
+public class ConnectionSingelton {
+    private static Connection connection = null;
+
+
+    public static Connection createConnection() {
```

Comment:
multiple threads may concurrently create multiple instances of singleton, only one thread should have ability to instantiate it